### PR TITLE
[3.2] Bump Quarkus version used by Test Suite in Github actions to 3.2.6

### PR DESCRIPTION
**Pull request by michalvavrik**
### Summary

TSIA

Please select the relevant options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Dependency update
- [ ] Refactoring
- [ ] Backport
- [ ] New scenario (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [ ] Methods and classes used in PR scenarios are meaningful
- [x] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
         <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
         <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
         <quarkus.platform.version>3.2.5.Final</quarkus.platform.version>
-        <quarkus.ide.version>3.2.5.Final</quarkus.ide.version>
+        <quarkus.ide.version>3.2.6.Final</quarkus.ide.version>
         <quarkus.qe.framework.version>1.3.0.Beta26</quarkus.qe.framework.version>
         <quarkus-qpid-jms.version>2.4.0</quarkus-qpid-jms.version>
         <apache-httpclient-fluent.version>4.5.14</apache-httpclient-fluent.version>
```
Comment (mocenas) [inline pom.xml:25]: Is there a reason why to change only quarkus IDE and not quarkus BOM? I would rather see this two changes together.

Comment (michalvavrik) [reply pom.xml:25]: Yeah, I actually meant to change platform version, thanks

Comment (michalvavrik) [reply pom.xml:25]: Done.

**Review by mocenas — APPROVED** [review]
LGTM

**Pull request merged**